apps/wing-console/console/server/src/utils/test-runner/simulator-manager.ts

```diff
@@ -0,0 +1,58 @@
+import type { Simulator } from "../../wingsdk.js";
+import { createCompiler, type Compiler } from "../compiler.js";
+import { createSimulator } from "../simulator.js";
+
+export const createSimulatorManager = ({
+  wingfile,
+  platform,
+  watchGlobs,
+}: {
+  wingfile: string;
+  platform?: string[];
+  watchGlobs?: string[];
+}) => {
+  let testCompiler: Compiler;
+  let simfilePath = "";
+
+  const createSimulatorInstance = async (simfile?: string) => {
+    if (!simfile && !simfilePath) {
+      throw new Error("No simfile provided");
+    } else if (!simfile) {
+      simfile = simfilePath;
+    }
```

Comment:
I believe you can get rid of the else here because the throw acts as an early return.